packages/lib/models/Resource.ts

```diff
@@ -637,7 +637,7 @@ export default class Resource extends BaseItem {
 		}
 
 		const output = await super.save(resource, options);
-		if (isNew) eventManager.emit(EventName.ResourceCreate);
+		eventManager.emit(isNew ? EventName.ResourceCreate : EventName.ResourceChange, { id: output.id });
```

Comment:
This pull request also causes `ResourceChange` events to be emitted when a resource's metadata changes. Previously, `ResourceChange` events were fired only [while resources were being watched](https://github.com/laurent22/joplin/blob/df3aaa7dfde2c4e6bbe1709721c07a2ad891e40c/packages/lib/services/ResourceEditWatcher/index.ts#L129).